source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3184,6 +3184,36 @@ Note that :mixxx:coref:`[Library],MoveUp` and other Move and Scroll controls emu
    .. versionadded:: 2.1.0
 
 
+.. mixxx:control:: [Library],MoveTrackUp
+
+   Move track selection up by one row in the playlist. Equivalent to pressing the :kbd:`Alt` + :kbd:`Up` key on the keyboard. This is applicable only in unlocked playlists.
+
+   :range: Binary
+   :feedback: Currently selected tracks are moved
+
+   .. versionadded:: 2.5.0
+
+
+.. mixxx:control:: [Library],MoveTrackDown
+
+   Move track selection down by one row in the playlist. Equivalent to pressing the :kbd:`Alt` + :kbd:`Down` key on the keyboard. This is applicable only in unlocked playlists.
+
+   :range: Binary
+   :feedback: Currently selected tracks are moved
+
+   .. versionadded:: 2.5.0
+
+
+.. mixxx:control:: [Library],MoveTrack
+
+   Move track selection up or down by one row in the playlist. Equivalent to pressing the :kbd:`Alt` + :kbd:`Up` or :kbd:`Alt` + :kbd:`Down` key on the keyboard. This is applicable only in unlocked playlists.
+
+   :range: Binary
```

Comment:
This should actually be [`integer` (move by n rows)](https://github.com/mixxxdj/mixxx/blob/4c11d9b538cf8d27acdef6665ce1b54658b7690d/src/library/librarycontrol.cpp#L824-L836), but I just noticed that this optional factor is not considered in [`WLibraryTableView::moveCursor()`](https://github.com/mixxxdj/mixxx/blob/a48562ad96021bd9571c8fbc8d331291bc30c1fc/src/widget/wlibrarytableview.cpp#L265).

I've just  filed https://github.com/mixxxdj/mixxx/issues/13398